OneSignalSDK/onesignal/core/src/main/java/com/onesignal/core/internal/operations/impl/OperationRepo.kt

```diff
@@ -121,12 +128,16 @@ internal class OperationRepo(
         queueItem: OperationQueueItem,
         flush: Boolean,
         addToStore: Boolean,
+        index: Int = -1,
```

Comment:
Using `null` is better practice over magic numbers (especially since Kotlin has null safety)